kubernetes/plausible.yaml

```diff
@@ -42,53 +42,6 @@ spec:
         runAsUser: 1000
         runAsGroup: 1000
         fsGroup: 1000
-      initContainers:
-        - name: plausible-init
-          image: plausible/analytics:latest
-          command:
-            - "/bin/sh"
-            - "-c"
-          args:
-            - sleep 30 && /entrypoint.sh db createdb && /entrypoint.sh db migrate && /entrypoint.sh db init-admin
-          envFrom:
-            - secretRef:
-                name: plausible-config
-          env:
-            - name: POSTGRES_USER
-              valueFrom:
-                secretKeyRef:
-                  name: plausible-db-user
-                  key: username
-            - name: POSTGRES_PASSWORD
-              valueFrom:
-                secretKeyRef:
-                  name: plausible-db-user
-                  key: password
-            - name: CLICKHOUSE_USER
-              valueFrom:
-                secretKeyRef:
-                  name: plausible-events-db-user
-                  key: username
-            - name: CLICKHOUSE_PASSWORD
-              valueFrom:
-                secretKeyRef:
-                  name: plausible-events-db-user
-                  key: password
-            - name: DATABASE_URL
-              value: postgres://$(POSTGRES_USER):$(POSTGRES_PASSWORD)@$(PLAUSIBLE_DB_SERVICE_HOST):$(PLAUSIBLE_DB_SERVICE_PORT)/plausible
-            - name: CLICKHOUSE_DATABASE_URL
-              value: http://$(CLICKHOUSE_USER):$(CLICKHOUSE_PASSWORD)@$(PLAUSIBLE_EVENTS_DB_SERVICE_HOST):$(PLAUSIBLE_EVENTS_DB_SERVICE_PORT)/plausible
-            - name: SMTP_HOST_ADDR
-              value: $(PLAUSIBLE_SMTP_SERVICE_HOST)
-          securityContext:
-            allowPrivilegeEscalation: false
-          resources:
-            limits:
-              memory: 2Gi
-              cpu: 1500m
-            requests:
-              memory: 50Mi
-              cpu: 10m
```

Comment:
Just removing the `initContainer` is insufficient. Maybe it's OK for docker-compose to manually initialize the admin, but I'd rather not `kubectl exec` into the plausible container to initialize the administrator account.